tetris.js

```diff
@@ -452,5 +452,47 @@ module.exports = {
 				return this.coordinates
 			}
 		}
+	},
+	Field: class {
+		constructor(field) {
+			if (Array.isArray(field)) {
+				this.field = field
+			} else {
+				this.field = []
+				for (let i = 0; i < field.y; i++) {
+					let array = []
+					for (let j = 0; j < field.x; j++) {
+						array.push(false)
+					}
+					this.field.push(array)
+				}
+			}
+		}
+		addTetrominoToField(coordinatesOfTetromino) {
+			for (let i = 0; i < coordinatesOfTetromino.length; i++) {
+				this.field[coordinatesOfTetromino[i].y][coordinatesOfTetromino[i].x] = true
+			}
+			return this.field
+		}
+		cleanFilledRows() {
+			let partOfTheField = []
+			for (let i = 0; i < this.field[0].length; i++) {
```

Comment:
you can save 
this.field[0].length
and
this.field.length
in separate field in the class. Initialize them in the constructor
Something like 
size: {x: ?, y: ?}